command/healthcheck/shared.go

```diff
@@ -32,24 +32,17 @@ func StringList(source interface{}) ([]string, error) {
 	return nil, fmt.Errorf("unknown source type for []string coercion: %T", source)
 }
 
-func fetchMountTune(e *Executor, versionError func()) (bool, *PathFetch, map[string]interface{}, error) {
+func fetchMountTune(e *Executor, versionError func()) (*PathFetch, error) {
 	tuneRet, err := e.FetchIfNotFetched(logical.ReadOperation, "/sys/mounts/{{mount}}/tune")
 	if err != nil {
-		return true, nil, nil, err
+		return nil, fmt.Errorf("failed to fetch mount tune information: %w", err)
 	}
 
 	if !tuneRet.IsSecretOK() {
 		if tuneRet.IsUnsupportedPathError() {
 			versionError()
 		}
-
-		return true, nil, nil, nil
```

Comment:
Note that removing this return `exit = true` isn't changing any behavior as both calling functions will simply return the returned err value if this is true which in this case is `nil`.  So because err == nil, each checker's `Evaluate` handler is called.

Reply:
Does make it a little inconsistent w.r.t. other helpers, but I think I'm fine with that given this lives in Shared versus the PKI which has more helpers aligning to that format.